Makefile

```diff
@@ -48,27 +56,25 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
-					   CONTIV_HOST_GOROOT=$(HOST_GOROOT) ./scripts/unittests -vagrant
+unit-test: build clean
```

Comment:
Because this situation happens with the system tests on linux:

* make build fires, builds on a vm, generates pkg dirs
* systemtests fire, which compile on the host, which may have an incompatible set of objects in pkg, and then it fails.